test/integration/targets/inventory/runme.sh

```diff
@@ -37,3 +37,9 @@ ANSIBLE_TRANSFORM_INVALID_GROUP_CHARS=never ansible-playbook -i ../../inventory
 
 # test extra vars
 ansible-inventory -i testhost, -i ./extra_vars_constructed.yml --list -e 'from_extras=hey ' "$@"|grep '"example": "hellohey"'
+
+# test constructed inventory
+ansible-playbook -i ./yaml_inventory.yml -i ./constructed.yml test_constructed_inventory.yml "$@"
```

Comment:
This is failing but the test script is missing `-e` so the test continues when it should fail.

Reply:
Good spotting, thanks.